src/main/webapp/app/stores/firebase/firebase-app.store.ts

```diff
@@ -8,7 +8,7 @@ import { notifyError } from 'app/oncokb-commons/components/util/NotificationUtil
 import { getAuth, onAuthStateChanged, signInWithCustomToken, signOut } from 'firebase/auth';
 import { AppConfig } from 'app/appConfig';
 
-export class FirebaseStore extends BaseStore {
+export class FirebaseAppStore extends BaseStore {
```

Comment:
Could you check if FirebaseAppStore still needs to extend BaseStore. If not, we can remove.

Reply:
It still needs to use the `readHandler` from BaseStore. Don't think we need to duplicate that logic, and I don't think it's an issue extending BaseStore, so I think we can probably keep it.